packages/kit/src/components/UpdateReminder/hooks.tsx

```
@@ -0,0 +1,351 @@
+        break;
+      default:
+        toDownloadAndVerifyPage();
+        break;
+    }
+  }, [
+    appUpdateInfo.status,
+    intl,
+    isFullModal,
+    toDownloadAndVerifyPage,
+    toUpdatePreviewPage,
+  ]);
```

ESLint

[error] React Hook useCallback has a missing dependency: 'showUpdateInCompleteDialog'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)